package.json

```diff
@@ -16,7 +16,9 @@
 		"onCommand:ops.runOpen",
 		"onCommand:ops.build",
 		"onCommand:ops.stop",
-		"onCommand:ops.runWithConfig"
+		"onCommand:ops.runWithConfig",
+		"onCommand:ops.runImage",
```

Comment:
Please, change the `runImage` and `stopImage` references to `startInstance/runInstance` and `stopInstance` to keep consistency with `ops` commands-

Reply:
Fixed by https://github.com/nanovms/ops-vscode/pull/10/commits/7448b1adb8c15a28cca930c76df3c354905f19cf